### PR TITLE
Add joined users

### LINKED ISSUE #84: Display names of members that joined room nicely
Display in both game master and member's screens

### PR DESCRIPTION
**Pull request by hzxcaryn**
Display connected players in create room page nicely

Closes #84

### CODE REVIEW

File: Web/src/components/common/VirtualizedList.tsx [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+import {
+  List,
+  Index,
+  WindowScroller,
+  ListRowRenderer,
+  AutoSizer,
+} from 'react-virtualized';
+
+type VirtualizedListProps = {
+  overscanRowCount?: number;
```
Comment (indocomsoft) [inline Web/src/components/common/VirtualizedList.tsx]: Seems unused?

File: Web/src/components/common/VirtualizedList.tsx [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+  noRowsRenderer?: () => JSX.Element;
```
Comment (indocomsoft) [inline Web/src/components/common/VirtualizedList.tsx]: Seems unused?

File: Web/src/components/common/VirtualizedList.tsx [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+  rowCount: number;
+  rowHeight:
```
Comment (indocomsoft) [inline Web/src/components/common/VirtualizedList.tsx]: Lol what's up with this typing

File: Web/src/components/common/VirtualizedList.tsx [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+  | number
+  | (number & ((info: Index) => number))
+  | (((params: Index) => number) & number)
+  | (((params: Index) => number) & ((info: Index) => number));
+  rowRenderer: ListRowRenderer;
+  scrollToIndex?: number;
+};
+
+const WindowScrollVirtualizedList: React.FC<VirtualizedListProps> = ({
```
Comment (indocomsoft) [inline Web/src/components/common/VirtualizedList.tsx]: Perhaps what you want is `import { ListProps } from 'react-virtualized';`
And then
```suggestion
const WindowScrollVirtualizedList: React.FC<ListProps> = ({
```

Or if you want to limit the keys, can use this https://www.typescriptlang.org/docs/handbook/utility-types.html#picktk

File: Web/src/components/create-room/RoomMembers.tsx [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+const createMembersRowRenderer = (users: string[]) => ({ key, index, style }: ListRowProps) => (
+  <div
+    key={key}
+    style={{
+      ...style,
+      display: 'flex',
+    }}
+  >
+    <MemberRow>
+      {users[users.length - 1 - index]}
```
Comment (indocomsoft) [inline Web/src/components/create-room/RoomMembers.tsx]: Hmm is this to render backwards? Might be clearer to do like

```typescript
(users: string[]) => {
  const reversedUsers = users.reverse();
  return ({ key, index, style }: ListRowProps) => (
  ...
  );
}

```

Comment (hzxcaryn) [reply Web/src/components/create-room/RoomMembers.tsx]: Yep to render backwards HAHA sure

**Pull request merged**